src/SFML/Audio/SoundStream.cpp

```diff
@@ -78,6 +78,8 @@ void SoundStream::initialize(unsigned int channelCount, unsigned int sampleRate)
 {
     m_channelCount = channelCount;
     m_sampleRate   = sampleRate;
+    m_samplesProcessed = 0;
+    m_isStreaming = false;
 
```

Comment:
I'm personally not in favour of too much alignment in the middle of lines (but yes, I was before, and I'm probably the one who wrote these two aligned lines).